### DISCUSSION TITLE
Allow adding custom endpoints to OAS spec

### DISCUSSION THREAD

**Discussion by HalikovStudioIT** [directus/directus#5745] [Feature Requests]
Hello. Is it possible to add custom endpoints to OAS? If not, are you planning to add such a feature?

**Comment by Eliophot** [comment]
Hello, I would like to repeat this subject, we have the same problem. Do we have to add something specific in the code so that the custom endpoints go back into the OpenAPI spec ?

**Reply by rijkvanzanten** [reply]
There's currently no way to inject custom values into the specs generated in the /specs endpoints. This post is an open feature request to add such functionality 👍🏻

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
